app/src/services/httpService.test.js

```diff
@@ -0,0 +1,66 @@
+import fetchMock from 'fetch-mock';
+import {HttpService} from './httpService';
+
+it('Test get method for response 200', () => {
```

Comment:
@ssmails, I'd say you don't need `Test` at the head of these desc strings

the test runner should make it clear enough where this is coming from, you should just be able to name the test as such, describing what the HttpService (module under test) is doing/should do

`it('returns 200 to GET request`,...)`

Reply:
@davemasselink Yes, done.